pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
+stdenv.mkDerivation rec {
+
+  pname = "LAStools-${version}";
+  version = "201003";
+
+  src = fetchFromGitHub {
+    owner = "LAStools";
+    repo = "LAStools";
+    rev = "635b76b42cc4912762da31b92f875df5310e1714";
+    sha256 = "0682ca3bp51lmfp46vsjnd1bqpn05g95pf4kclvjv1y8qivkxsaq";
+  };
+
+  patches = [
+    ./drop-64-suffix.patch
```

Comment:
A comment as to why this patch is required would be nice.

Reply:
fixed in 29ab7eaacd27d06c235b811114ed12374c6d68e0